lib/aptible/cli/resource_formatter.rb

```diff
@@ -9,13 +9,28 @@ def inject_account(node, account)
           node.value('handle', account.handle)
         end
 
+        def inject_operation(node, operation)
+          node.value('id', operation.id)
+          node.value('status', operation.status)
+          node.value('git_ref', operation.git_ref)
+          node.value('docker_ref', operation.docker_ref)
```

Comment:
This is practically always going to be `null` (this field might still be used on the handful of remaining deprecated v1 stacks, but it's not used anywhere else today), so I'm pretty certain this will not do what you want it to do.

What's the purpose of including this here?

Reply:
You're right - I misinterpreted what the attribute was used for. Removed.